### ISSUE TITLE
Storybook fails to start locally

### ISSUE THREAD

**Issue by noodny** [nearform/titus#1472]
This came up while working on https://github.com/nearform/titus/pull/1471.

Storybook fails to start due to a bug in a release of [react-refresh](https://www.npmjs.com/package/react-refresh), which is being used by [@storybook/preset-create-react-app](https://www.npmjs.com/package/@storybook/preset-create-react-app). Due to a fix not being published for a couple of months, the most reasonable way to use the fixed version is to [update react-scripts to 5.0](https://github.com/nearform/titus/pull/1443). This will however require storybook itself to be updated.

TLDR - update everything that's storybook related

**Comment by Eomm** [comment]
Sidenote: updating react-scripts to 5, should unblock #923: now the installation fails due d conflict generated by `react-scripts@4`

**Comment by simoneb** [comment]
@Eomm I tried to do it a few days ago on a PR that dependabot had opened and I wished I hadn't tried. I found myself in a vale of tears
